src/less/radio/ds6/radio.less

```diff
@@ -11,6 +11,16 @@
     width: 24px;
 }
 
+@media (prefers-color-scheme: dark) {
```

Comment:
These custom props should be scoped to `.radio` not `.radio__checked` and `.radio__unchecked`.

Don't forget to also add the default custom properties that sit outside of this dark mode media query. Also, please add a screenshot of the module in dark mode to the PR (you might need to temporarily add `skin-experiment-1` class to the HTML body).

Reply:
Fixed!